lib_test/bap_project/test_project.ml

```diff
@@ -5,20 +5,26 @@ open Format
 open Word_size
 open Bap.Std
 
+type cmp =
+  | Precise
+  | Regexpr 
+
 type case = {
   arch : arch;
   addr : int;
   code : string;
   bil : string;
   asm : string;
+  cmp : cmp;
 }
 
 let arm = {
   arch = `armv7;
   addr = 16;
   code = "\x00\x20\xA0\xE3";
   bil  = "R2 := 0x0:32";
-  asm  = "mov r2, #0x0";
+  asm  = "mov r2, #(0x0|0)$";
```

Comment:
fix the instruction to use registers instead of the numerals. The test is not for verifying that we can print numbers, 
but for verifying that we can annotate project structure.